apps/src/lab2/levelEditors/aichatSettings/UpdateContext.ts

```diff
@@ -20,4 +20,8 @@ export const UpdateContext = createContext({
     property: keyof ModelCardInfo,
     value: ModelCardInfo[keyof ModelCardInfo]
   ) => {},
+  setModelSelectionValues: (
+    availableModelIds: string[],
+    selectedModelId: string = ''
```

Comment:
context newbie...what does this default value do?

Reply:
I'm not sure if I need it in both places, but this was a default for that parameter. I actually meant to remove that.

Reply:
Oh and this is just a default value so they're empty functions I guess.